awx/ui_next/package.json

```diff
@@ -55,6 +55,7 @@
     "react-scripts": "^3.4.4"
   },
   "scripts": {
+    "prestart": "lingui compile",
```

Comment:
This will automatically be run when `npm start` is called, which will compile the strings and create the necessary `messages.js` files.